cross/tcl/PLIST

```diff
@@ -0,0 +1,257 @@
+rsc:bin/sqlite3_analyzer
+bin:bin/tclsh8.6
+rsc:lib/itcl4.0.5/itclConfig.sh
+rsc:lib/itcl4.0.5/itclHullCmds.tcl
+rsc:lib/itcl4.0.5/itcl.tcl
+rsc:lib/itcl4.0.5/itclWidget.tcl
+rsc:lib/itcl4.0.5/libitcl4.0.5.so
+rsc:lib/itcl4.0.5/pkgIndex.tcl
+lib:lib/libtcl8.6.so
+rsc:lib/pkgconfig/tcl.pc
+lib:lib/sqlite3.13.0/libsqlite3.13.0.so
+rsc:lib/sqlite3.13.0/pkgIndex.tcl
+rsc:lib/tcl8.6/auto.tcl
+rsc:lib/tcl8.6/clock.tcl
+rsc:lib/tcl8.6/encoding/ascii.enc
```

Comment:
Minimize file size by using `rsc:lib/tcl8.6/encoding`, and removing the subsequent entries untl L92